frontend/src/Calendar/iCal/CalendarLinkModalContent.js

```diff
@@ -140,7 +140,7 @@ class CalendarLinkModalContent extends Component {
                 type={inputTypes.CHECK}
                 name="premieresOnly"
                 value={premieresOnly}
-                helpText={translate('ICalSeasonPremieresOnlyHelpText')}
+                helpText={translate('ICalSeasonPremiereEpisodesOnlyHelpText')}
```

Comment:
This one should already be unique to Sonarr.

Reply:
reverted, missed that it already had Season in!